taskcluster/kinds/update-db/kind.yml

```diff
@@ -24,7 +24,8 @@ tasks:
       substitution-fields:
         - attributes
         - run.command
-    run-on-tasks-for: []
+    # todo: replace to []
+    run-on-tasks-for: ["github-pull-request"]
```

Comment:
> Thought 

This commit doesn't make sense to me.

Reply:
This was just for testing, to trigger it on every push in a pull request. It will run as a cron job. Also, this PR was not meant to be reviewed commit by commit.